lambda/authorizer/lambda_functions.py

```diff
@@ -134,3 +143,30 @@ def is_admin(jwt_data: dict[str, Any], admin_group: str, jwt_groups_property: st
         else:
             return False
     return admin_group in current_node
+
+
+def refresh_management_tokens() -> list[str]:
+    """Return secret management tokens if they exist."""
+    global secret_tokens
```

Comment:
lets rename to `get_management_tokens` and then remove the global variable since we don't have a "global" use case based on the code here